tools/compiler/main.rs

```diff
@@ -29,6 +29,10 @@ struct Cli {
     #[arg(name = "dependency file", long = "depfile", number_of_values = 1, action)]
     depfile: Option<std::path::PathBuf>,
 
+    /// Embed resources
+    #[arg(long, name = "bool")]
+    embed_resources: Option<bool>,
```

Comment:
A true equivalent of the corresponding Rust API would be to take a string, I think. We could still accept "true"/"false", but AFAICS there are four possible values in total: Three variants of https://slint-ui.com/releases/0.3.5/docs/rust/slint_build/enum.embedresourceskind and the case of "don't embed".

Reply:
i think so, we need it for https://github.com/slint-ui/slint/issues/2136 , but the embedded for software renderer part is only partially implemented (no fonts yet)